backend/api/Areas/Tools/Helpers/ImportProjectsHelper.cs

```diff
@@ -108,45 +140,117 @@ private Entity.Project Merge(Entity.Project project, Model.ImportProjectModel mo
-                project.SalesCost = model.SalesCost;
-                project.InterestComponent = model.InterestComponent;
-                project.NetProceeds = model.NetProceeds;
-                project.GainLoss = model.GainLoss;
-                project.SaleWithLeaseInPlace = model.SaleWithLeaseInPlace;
-
-                if (model.PriorNetProceeds.HasValue)
-                {
-                    project.Snapshots.Add(new Entity.ProjectSnapshot(project) { NetProceeds = model.PriorNetProceeds.Value });
-                }
+                project.Market = model.Market;
+                project.Appraised = model.Appraised;
 
-                project.MarketedOn = model.MarketedOn;
+                project.SubmittedOn = new DateTime(model.ReportedFiscalYear - 1, 1, 1); // Don't have a source for this information.
+                project.ApprovedOn = project.SubmittedOn; // Don't have a source for this information.
                 project.CompletedOn = model.CompletedOn;
 
-                project.PrivateNote = model.PrivateNote;
+                var metadata = new Entity.Models.DisposalProjectSnapshotMetadata
+                {
+                    InitialNotificationSentOn = null, // Don't have a source for this information.
+                    ThirtyDayNotificationSentOn = null, // Don't have a source for this information.
+                    SixtyDayNotificationSentOn = null, // Don't have a source for this information.
+                    NinetyDayNotificationSentOn = null, // Don't have a source for this information.
+                    OnHoldNotificationSentOn = null, // Don't have a source for this information.
+                    InterestedReceivedOn = null, // Don't have a source for this information.
+                    TransferredWithinGreOn = null, // Don't have a source for this information.
+                    ClearanceNotificationSentOn = null, // Don't have a source for this information.
+                    RequestForSplReceivedOn = model.RequestForSplReceivedOn, // Don't have a source for this information.
+                    ApprovedForSplOn = model.ApprovedForSplOn,
+                    MarketedOn = model.MarketedOn,
+                    Purchaser = model.Purchaser,
+                    OcgFinancialStatement = model.OcgFinancialStatement,
+                    AppraisedBy = model.AppraisedBy,
+                    ProgramCost = model.ProgramCost,
+                    SalesCost = model.SalesCost,
+                    InterestComponent = model.InterestComponent,
+                    NetProceeds = model.NetProceeds,
+                    GainLoss = model.GainLoss,
+                    SaleWithLeaseInPlace = model.SaleWithLeaseInPlace,
+                    DisposedOn = model.DisposedOn ?? (project.Status.Code == "DIS" ? model.CompletedOn : null),
+                    IsContractConditional = model.IsContractConditional.HasValue ? model.IsContractConditional : false, // Don't have a source for this information.
```

Comment:
where is model.iscontractConfitional coming from then?

Reply:
I could add it to the JSON sent to the import, but SIS data doesn't have the data.